tests/test_chaining.py

```diff
@@ -0,0 +1,6 @@
+import os
+import pytest
+
+import toolchest_client as toolchest
+
+THIS_DIRECTORY = os.path.dirname(os.path.realpath(__file__))
```

Comment:
Is this here as a placeholder for integration tests? Are you planning to add before you merge?

Reply:
Precisely; I'll add tests in a follow-up commit to this PR.